.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 4,
+    "true": false,
+    "singleQuote": false,
+    "printWidth": 140
```

Comment:
My $.02, I prefer something wider than 80. The guidelines didn’t give a strong reason to prefer 80, IMO, and I often find 80 to be too restrictive.

Reply:
I went with 140 because that seemed to be the length of the longest lines I was finding in the codebase, and figured it'd result in the lease number of whitespace changes.

I'd prefer to reduce it to 100, just my preference in width.